packages/bridge/src/components/CurrentUserWalletBadge/index.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 
-import { useWallet, WALLET_PROVIDERS } from '@oyster/common';
 import { shortenAddress } from '@oyster/common';
+import { useWallet } from '@solana/wallet-adapter-react';
 
 export const CurrentUserWalletBadge = (props: { showDisconnect?: boolean }) => {
-  const { wallet, disconnect } = useWallet();
+  const { wallet, publicKey, disconnect } = useWallet();
```

Comment:
Hmm. I would hope not to change this in the adapter interface itself or in the `react` package because `wallet.publicKey` is used in a bunch of different codebases in the wild, and this is easy to port over to right now with
```
const wallet = useWallet(); // can use wallet.publicKey now
```

What about just aliasing it here and where needed as `const { publicKey: walletPublicKey } = useWallet();`?

Reply:
If it's difficult to change in the api then let's leave it as is. My guess is  most people won't alias it so we most likely  would only add more inconsistency.